CurrencyConverterControl/ComboboxValueConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace CurrencyConverterControl
+{
+    internal class ComboboxValueConverter : IValueConverter
```

Comment:
all the classes which are used from xaml, should be public. If they are internal, people won't be able to re-use your xaml to create own control template

Reply:
> all the classes which are used from xaml, should be public. If they are internal, people won't be able to re-use your xaml to create own control template Also, as I see this converter doesn't do anything interesting, maybe it's not required. Do you see any problems when you remove it from ComboBox bindings?

Yeh the Converter in fact helps me to give a Selected Value to the Combobox, predefined from the Main application. If i do remove it, the valuegiven into the MainWindow wont be given to combobox.